qemu/tests/numa_consistency.py

```diff
@@ -91,9 +91,9 @@ def get_vcpu_used_node(numa_node_info, vcpu_thread):
         memory_status, _ = utils_test.qemu.get_numa_status(host_numa_node,
                                                            qemu_pid)
         memory_used_after = memory_status[node_used_host_index]
-
-        memory_allocated = (memory_used_after - memory_used_before) * 4 / 1024
-
+        pa = int(params.get("page_size"))
```

Comment:
I prefer to get this value via python api
https://docs.python.org/2/library/resource.html#resource.getpagesize